worker/src/main/resources/application.properties

```diff
@@ -25,17 +25,21 @@ eob.job.patient.queue.max.size=10000
 # Number of patients to load into queue at one time
 eob.job.patient.queue.page.size=1000
 
+
+## ----------------------
+
+# Amount of time to wait to write to a file before throwing an error
 file.try.lock.timeout=30
-cancellation.check.frequency=1000
+
+# How often to write out % complete to the database (for display in status call to API)
 report.progress.db.frequency=100
+
+# How often to write out % complete to stdout
```

Comment:
Put in the comment what the units are.  Are these msec or something else?

As a future aside, having the name of the property include the units is much clearer.  e.g. report.progress.db.frequency.ms=